Flaky: docker daemon not started

In the CI, the buildbot container occasionally fails to contact the docker daemon. This could be the result of our check for docker-in-docker container start being bad (`docker --version` at the moment), or another issue entirely.

Wait step:
```
bash -c '
for i in {1..10}
do
  docker --version &> /dev/null && exit
  sleep 2
done
echo "Could not reach Docker daemon from buildbot worker" >&2
exit 1'
[...]
program finished with exit code 0
elapsedTime=0.126445
```

Build step:
```
Cannot connect to the Docker daemon at tcp://localhost:2375. Is the docker daemon running?
make: *** [/home/eve/workspace/kube_pod-prod-metalk8s-backend-0/build/_build/root/salt/metalk8s/containerd/files/pause-3.1.tar] Error 1
```

--- a/eve/main.yml
+++ b/eve/main.yml
@@ -40,7 +40,7 @@ stages:
           name: Wait for Docker daemon to be ready
           command: |
             bash -c '
-            for i in {1..10}
+            for i in {1..150}
             do
               docker info &> /dev/null && exit
               sleep 2

